src/lib.rs

```diff
@@ -43,6 +43,8 @@ pub mod fcntl;
           target_os = "openbsd"))]
 pub mod ifaddrs;
 #[cfg(any(target_os = "linux", target_os = "android"))]
+pub mod kmod;
```

Comment:
Not sure if this is the right place to put this nor if this is the proper name. Inputs are welcome.